api/v1beta1/hyperconverged_types.go

```diff
@@ -466,6 +466,7 @@ type HyperConvergedFeatureGates struct {
 	// +optional
 	// +kubebuilder:default=false
 	// +default=false
```

Comment:
Please remove the default value lines. Also, please remove the default from line 69 & 851

Reply:
Please also remove the detailed documentation for this field. if it's ignored, documenting it may be confusing

Reply:
Done :)